### PR TITLE
WFP-1428

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/listener/ExtractPlacedEventListenerTestTest.kt [hunk 1]
```diff
@@ -14,7 +14,9 @@ import uk.gov.justice.digital.hmpps.hmppsworkload.integration.jpa.entity.Reducti
 import uk.gov.justice.digital.hmpps.hmppsworkload.integration.jpa.entity.ReductionReasonEntity
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/listener/ExtractPlacedEventListenerTestTest.kt:14]: this file name ends in TestTest

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/OffenderManagerEntity.kt [hunk 2]
```diff
@@ -165,6 +167,7 @@ data class OffenderManagerEntity(
   @Column
   val surname: String,
 
-  @Column
-  val typeId: Long
+  @OneToOne
+  @JoinColumn(name = "type_id")
+  val offenderManagerType: OffenderManagerTypeEntity
```
Comment (sp-watson) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/OffenderManagerEntity.kt:172]: is this 1-to-1? I would have thought there would be 1 type for many managers?!

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/OffenderManagerTypeEntity.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+@Entity
+@Table(name = "offender_manager_type", schema = "app")
+data class OffenderManagerTypeEntity(
+  @Id
+  @Column
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  val gradeCode: String
```
Comment (sp-watson) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/OffenderManagerTypeEntity.kt:18]: So the offender manager type is defined by the grade and the grade only?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/OffenderManagerTypeEntity.kt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "offender_manager_type", schema = "app")
```
Comment (sp-watson) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/OffenderManagerTypeEntity.kt:11]: This table already exists then?

**Pull request closed without merging**